src/resources/projects/website/templates/sidebar.ejs

```diff
@@ -36,7 +36,7 @@
     <div class="pt-lg-2 mt-2 <%= alignCss %> sidebar-header<%= sidebar.logo && sidebar.title ? ' sidebar-header-stacked' : '' %>">
     <% if (sidebar.logo) { %>
       <a href="<%- sidebar['logo-href'] || '/index.html' %>" class="sidebar-logo-link">
-      <img src="<%- sidebar.logo %>" alt="<%- navbar['logo-alt'] || '' %>" class="sidebar-logo py-0 d-lg-inline d-none"/>
+      <img src="<%- sidebar.logo %>" alt="<%- sidebar['logo-alt'] || '' %>" class="sidebar-logo py-0 d-lg-inline d-none"/>
```

Comment:
Oh, that's a nice catch. Thanks.

We should add a regression test as well.

Reply:
Not sure what you want to test exactly but the issue was clearly a copy/paste issue from the navbar implementation that was not updated properly.

I can add a test to check that alt is set in HTML when set in YAML.
While at it, I can add the same for the navbar.

Reply:
dd805e1